prov/rxm/src/rxm.h

```diff
@@ -186,6 +189,27 @@ struct rxm_pkt {
 	char data[];
 };
 
+union rxm_sar_ctrl_flags {
+	enum rxm_sar_seg_type {
+		RXM_SAR_SEG_FIRST	= 1,
+		RXM_SAR_SEG_MIDDLE	= 2,
+		RXM_SAR_SEG_LAST	= 3,	
```

Comment:
LAST sets 2 bits, which ends up not treating the field as flags.  Maybe the field should just rename to ctrl_data? or seg_info/data?

Reply:
Sure, this sounds reasonable